content/events/2020-02-28.md

```diff
@@ -1,12 +1,14 @@
 ---
 title: "Conference Automotive Software Strategies"
-date: 2020-02-28T16:27:43+00:00
```

Comment:
The prefered date format is 2020-02-28T16:27:43+00:00 not 2020-03-03

Reply:
I updated the dates to match this format.